config/develop/rna-seq-reprocessing-output-bucket.yaml

```diff
@@ -0,0 +1,36 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: rna-seq-reprocessing-toil-cluster-v001-out
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: 'tess.thyer@sagebase.org'
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  SynapseUserName: 'wpoehlm'
+  # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
+    - 'arn:aws:iam::563295687221:user/tess.thyer@sagebase.org'
+    - 'arn:aws:iam::563295687221:user/william.poehlman@sagebase.org'
+
+  # Restrict downloading files from this bucket to only AWS resources (e.g. EC2 , Lambda)
+  # within the same region as this bucket.  This will not allow even the owner of the bucket
+  # to download objects in this bucket when not using an AWS resource in the same region!
+  SameRegionResourceAccessToBucket: "true"
+  BucketName: "rna-seq-reprocessing-toil-cluster-v001-out"
```

Comment:
for consistency, should this be `rna-seq-reprocessing-sandbox-toil-cluster-v001-out` instead?

Reply:
ideally yes but that's not how it works in sandbox currently, and changing names in the current stack is non-trivial (one of many things needing refactor)